src/System.Threading.Overlapped/tests/ThreadPoolBoundHandle_Helpers.cs

```diff
@@ -19,14 +20,14 @@ struct NonBlittableType
         public string s;
     }
 
-    private static ThreadPoolBoundHandle CreateThreadPoolBoundHandle()
+    private ThreadPoolBoundHandle CreateThreadPoolBoundHandle()
     {
         return CreateThreadPoolBoundHandle((SafeHandle)null);
     }
 
-    private static ThreadPoolBoundHandle CreateThreadPoolBoundHandle(SafeHandle handle)
+    private ThreadPoolBoundHandle CreateThreadPoolBoundHandle(SafeHandle handle)
```

Comment:
> Any reason why we decided to not have them be static?

Because the implementation is based on the base class creating a temporary directory in which the test file path lives and then the disposal of that base class deleting the directory.